crates/dbs-virtio-devices/src/lib.rs

```diff
@@ -180,6 +180,9 @@ pub enum Error {
     /// Inserting mmap region failed.
     #[error("inserting mmap region failed: {0}")]
     InsertMmap(vm_memory::mmap::Error),
+    /// Failed to set madvise on guest memory region.
+    #[error("failed to set madvice() on guest memory region")]
+    Madvise(#[source] nix::Error),
```

Comment:
Not seeing any place to use this error type, will be introduced later?

Reply:
Due to some historical reason, dragonball device manager will use IO error, and it should have a new type for Madvise.